makeCall.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * Creates an endpoint that can be used in your TwiML App as the Voice Request Url.
+ *
+ * In order to make an outgoing call using Twilio Voice SDK, you need to provide a
+ * TwiML App SID in the Access Token. You can run your server, make it publicly
+ * accessible and use `/makeCall` endpoint as the Voice Request Url in your TwiML App.
+ */
+include('./vendor/autoload.php');
+include('./config.php');
+
+$callerId = 'client:quick_start';
+$to = isset($_POST["to"]) ? $_POST["to"] : NULL;
+if (empty($to) || !isset($to)) {
```

Comment:
Usually you want to check for isset before you check for empty. Is that true for PHP?